source/core/Ky.ts

```diff
@@ -365,4 +383,99 @@ export class Ky {
 			},
 		);
 	}
+
+	// eslint-disable-next-line @typescript-eslint/ban-types
+	protected _getTotalBytes(body?: globalThis.BodyInit | null): number {
+		if (!body) {
+			return 0;
+		}
+
+		if (body instanceof globalThis.FormData) {
+			// This is an approximation, as FormData size calculation is not straightforward
+			let size = 0;
+			// eslint-disable-next-line unicorn/no-array-for-each -- FormData uses forEach method
+			body.forEach((value: globalThis.FormDataEntryValue, key: string) => {
+				if (typeof value === 'string') {
+					size += new globalThis.TextEncoder().encode(value).length;
+				} else if (typeof value === 'object' && value !== null && 'size' in value) {
+					// This catches File objects as well, as File extends Blob
+					size += (value as Blob).size;
+				}
+
+				// Add some bytes for field name and multipart boundaries
+				size += new TextEncoder().encode(key).length + 40; // 40 is an approximation for multipart overhead
+			});
+
+			return size;
+		}
+
+		if (body instanceof globalThis.Blob) {
+			return body.size;
+		}
+
+		if (body instanceof globalThis.ArrayBuffer) {
+			return body.byteLength;
+		}
+
+		if (typeof body === 'string') {
+			return new globalThis.TextEncoder().encode(body).length;
+		}
+
+		if (body instanceof URLSearchParams) {
+			return new globalThis.TextEncoder().encode(body.toString()).length;
+		}
+
+		if ('byteLength' in body) {
+			return (body).byteLength;
+		}
+
+		if (typeof body === 'object' && body !== null) {
+			try {
+				const jsonString = JSON.stringify(body);
+				return new TextEncoder().encode(jsonString).length;
+			} catch (error) {
+				console.warn('Unable to stringify object:', error);
```

Comment:
As a library, it's generally considered bad practice for Ky to output non-errors to the console by default.

Reply:
removed